webapp/src/index.tsx

```diff
@@ -62,7 +60,7 @@ export default class Plugin {
         registry.registerAdminConsoleCustomSetting('ConnectedUsersReportDownload', ListConnectedUsers);
 
         // let settingsEnabled = (state as any)[`plugins-${manifest.id}`]?.connectedStateSlice?.connected || false; //TODO use connected selector from https://github.com/mattermost/mattermost-plugin-msteams/pull/438
-        let settingsEnabled = true;
+        let settingsEnabled = false;
```

Comment:
Not 100% sure whether this needs to be toggled or not

Reply:
Thanks. Nice catch...that was from testing...